mps_youtube/config.py

```diff
@@ -345,6 +345,7 @@ class _Config:
             ConfigItem("autoplay", False),
             ConfigItem("set_title", True),
             ConfigItem("mpris", not mswin),
+            ConfigItem("show_qrcode", True),
```

Comment:
I think the default value should be `False`. Any arguments to convince me it should be true?

Reply:
I considered making it `False`; my concern is that the feature is not very discoverable. Then again, I guess other features are as well.

Reply:
I agree :slightly_smiling_face: I've changed it!

Reply:
(Thanks for your help, btw!)